### PR TITLE
[Enchancement](mysql-compatable) add regression-test for MySQLdump

### PR DESCRIPTION
**Pull request by Cai-Yao**
# Proposed changes

Issue Number: close #xxx

## Problem summary

add regression-test for like this:
`mysqldump -h127.0.0.1 -P9030 -uroot --no-tablespaces --databases > /backup/mysqldump/test.db`

- To prevent errors `Unknown table 'column_statistics' in information_schema (1109)`, the table information_schema.column_statistics was added.

## Checklist(Required)

* [ ] Does it affect the original behavior
* [ ] Has unit tests been added
* [ ] Has document been added or modified
* [ ] Does it need to update dependencies
* [ ] Is this PR support rollback (If NO, please explain WHY)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: gensrc/thrift/Descriptors.thrift [hunk 1]
```diff
@@ -100,6 +100,7 @@ enum TSchemaTableType {
     SCH_SESSION_STATUS,
     SCH_SESSION_VARIABLES,
     SCH_STATISTICS,
+    SCH_COLUMN_STATISTICS,
```
Comment (yiguolei) [inline gensrc/thrift/Descriptors.thrift:103]: do not insert enum type in the middle of the definition. During upgrade, we will upgrade BE First and then upgrade FE, in this case,  the enum id in the definition is different. I will core.

**Review by yiguolei — APPROVED** [review]
LGTM

**Comment by Cai-Yao** [comment]
run buildall

**Review by yiguolei — APPROVED** [review]
LGTM

**Pull request merged**